test/snapshots/index.test.js

```diff
@@ -0,0 +1,80 @@
+const { test } = require("node:test");
```

Comment:
shouldnt this test be in the root ? also why remove white space? the whole point is to check for whitespaces

Reply:
I thought we were gonna keep the snapshots tests separate

Reply:
The test should stay in the test folder, snapshot output in the test/snapshot folder

Reply:
I tried to fix it